tox.ini

```diff
@@ -22,15 +19,19 @@ whitelist_externals =
     env
 
 [testenv:server]
-usedevelop = true
 passenv = PBENCH_UNITTEST_SERVER_MODE
+deps =
+   -r{toxinidir}/server/requirements.txt
+   -r{toxinidir}/test-requirements.txt
 commands_pre = 
   find {toxinidir} -type f -not -path '{toxinidir}/.tox/*' -path '*/__pycache__/*' -name '*.py[c|o]' -delete
 commands =
   bash -c "./server/bin/unittests {posargs}"
 
 [testenv:agent]
-usedevelop = true
+deps =
+   -r{toxinidir}/test-requirements.txt
+   -r{toxinidir}/agent/test-requirements.txt
```

Comment:
Does the ordering of these lines matter?  (These are reversed relative to 24-25.)

Reply:
Sort of reversed:  The server has its own `requirements.txt` file, and then uses the "common" `test-requirements.txt` file, while the agent does not have a `requirements.txt` file, but does have its own `test-requirements.txt` file now (newly added).  So I was using the order of "common first, specific second" for the agent "test" requirements.  In the case of the server it is just the ordering of the "code" requirements followed by the "test" requirements.

The question is: "why make a change to the agent side now"?

There is no Python 3 code any where in the tree, agent or server side, except a `pprof` tool which requires it.  But that is actually a mistake for the agent to require the Python 3 `ansible` module since it is only used by a tool.  We have a rule that the `pbench-agent` RPM does list RPM dependencies (or Python 3 dependencies, for that matter).

However, because this is a Python 3 module in `requirements.txt`, and the `pbench-agent` RPM does not currently install any modules via `pipe` during RPM installation on the `b0.69` branch (it only removes old `configtools` modules), there is no harm done with these changes.

So the change here is to reflect reality: we need the `ansible` module in our test environment, but not for the operation of the agent code.